templates/field/field--node--oe-social-media-links--oe-event--full.html.twig

```diff
@@ -6,8 +6,4 @@
  * @see ./core/themes/stable/templates/field/field.html.twig
  */
 #}
-<div class="ecl-u-mt-l">
```

Comment:
Class `ecl-u-mt-l` exists in modules/oe_theme_helper/templates/oe-theme-helper-social-media-links.html.twig